backend/MoleculeOSSite/Entities/User.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace MoleculeOSSite.Entities
+{
+    public class User
+    {
+        [Key]
+        public int Id { get; set; }
+        
+        public string Username { get; set; }
+        
+        public string Email { get; set; }
+        public DateTime JoinDate { get; set; }
+        
+        public string PasswordHash { get; set; }
+        [NotMapped]
+        public string ConfirmPassword { get; set; }
+
+        public int RoleId { get; set; } = 1;
+        public virtual Role Role { get; set; }
```

Comment:
```suggestion
namespace MoleculeOSSite.Entities
{
    public class User
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public DateTime JoinDate { get; set; }
        public string PasswordHash { get; set; }
        public string ConfirmPassword { get; set; }

        public int RoleId { get; set; } = 1;
        public Role Role { get; set; }
```
1. Wiadomo trzeba usunąć niepotrzebne rzeczy (Key i puste linie).
2. Nie wiem czy nie lepiej byłoby po prostu dać "Name" zamiast "Username" (ale to już jak uważasz).
3. "RoleId = 1" nic mi nie mówi. To jest rola admina czy usera?
4. Zamiast NotMapped można w MyDbContext to skonfigurować przy użyciu "Fluent API" z ef, ale w zasadzie najlepiej byłoby ConfirmPassword przenieść gdzie indziej.
5. virtual ci nie jest potrzebne, bo lazy loading nie jest nam potrzebny (z resztą w AccountService korzystasz z Include, więc tym bardziej dowodzi to, że trzeba to wyrzucić). Poza tym do skonfigurowania lazy loadingu to mało.